grafana-plugin/src/models/alert_receive_channel/alert_receive_channel.ts

```diff
@@ -105,23 +105,60 @@ export class AlertReceiveChannelStore {
     return alertReceiveChannel.data;
   }
 
-  async fetchServiceNowListOfStatus(): Promise<void> {
-    this.serviceNowStatusList = [
-      {
-        id: 1,
-        name: 'Resolved',
-      },
-      {
-        id: 2,
-        name: 'In Progress',
-      },
-      {
-        id: 3,
-        name: 'New',
-      },
-    ];
+  async fetchServiceNowStatusList({
+    id,
+    skipErrorHandling,
+  }: {
+    id: ApiSchemas['AlertReceiveChannel']['id'];
+    skipErrorHandling?: boolean;
+  }): Promise<void> {
+    const statusList = await onCallApi({ skipErrorHandling }).GET('/alert_receive_channels/{id}/status_options/', {
+      params: { path: { id } },
+    });
 
-    return Promise.resolve();
+    runInAction(() => {
+      // @ts-ignore // looks like wrong schema
+      this.serviceNowStatusList = statusList.data;
+    });
+  }
+
+  async hasServiceNowToken({ id }: { id: ApiSchemas['AlertReceiveChannel']['id'] }) {
```

Comment:
it should either update some observable instead of returning data from this store or should be moved to alert_receive_channel.helpers.ts (and actually we already have it there named `checkIfTokenExists`)